### PR TITLE
FW-4133 Add Sentry Integration

### PR DESCRIPTION
**Pull request by NolanVH**
### Description of Changes
This PR adds the Sentry SDK to the project, which will send information to the Sentry dashboard (links in Jira ticket).

### Checklist
- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [x] Tests have been updated / created if applicable
- [x] Admin Panel has been updated if models have been added or modified
- [x] Migrations have been updated and committed if applicable

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

File: firstvoices/firstvoices/settings.py [hunk 1]
```diff
@@ -196,3 +200,31 @@
...
 ELASTICSEARCH_PRIMARY_INDEX = os.getenv("ELASTICSEARCH_PRIMARY_INDEX", "fv")
+
+# Sentry monitoring configuration settings.
+# See docs at https://docs.sentry.io/platforms/python/guides/django/
+sentry_logging = LoggingIntegration(
+    level=logging.INFO,  # The minimum logging level to capture as breadcrumbs
+    event_level=logging.ERROR,  # The minimum logging level to send as events
+)
+sentry_sdk.init(
+    dsn=os.getenv("SENTRY_DSN"),
```
Comment (sarahfirstvoices) [inline firstvoices/firstvoices/settings.py:211]: Please add these new environment vars to the readme.

Comment (sarahfirstvoices) [reply firstvoices/firstvoices/settings.py:211]: What happens if they are not set (e.g., locally)? Do they need a default or do they fail gracefully?

File: firstvoices/firstvoices/settings.py [hunk 2]
```diff
@@ -196,3 +200,31 @@
...
+    environment=os.getenv(
+        "SENTRY_ENVIRONMENT"
+    ),  # Sends information to this environment on the dashboard
+    release=os.getenv("SENTRY_RELEASE"),  # Tags information with this release version
+    traces_sample_rate=1.0,  # The percentage of traces to send to sentry (min 0.0, max 1.0)
```
Comment (sarahfirstvoices) [inline firstvoices/firstvoices/settings.py:216]: Sample rate is another one that would be good to extract to an env variable, so we can change it in production if it gets too noisy.

**Review by sarahfirstvoices — APPROVED** [review]
I'm marking this approved because my comments are minor. No need to re-review after those changes

File: firstvoices/firstvoices/settings.py (hunk 2 above)
Comment (NolanVH) [reply firstvoices/firstvoices/settings.py:216]: I have made this an environment variable that will fall back to 1.0 if not set.

**Pull request merged**